Signal/src/ViewControllers/SettingsTableViewController.m

```diff
@@ -15,27 +15,32 @@
 #import "PrivacySettingsTableViewController.h"
 #import "PushManager.h"
 #import "Signal-Swift.h"
+#import "BlockListViewController.h"
 
 #define kProfileCellHeight 87.0f
 #define kStandardCellHeight 44.0f
 
 #define kNumberOfSections 4
 
 #define kRegisteredNumberRow 0
-#define kInviteRow 0
-#define kPrivacyRow 1
-#define kNotificationRow 2
-#define kLinkedDevices 3 // we don't actually use this, instead we segue via Interface Builder
-#define kAdvancedRow 4
-#define kAboutRow 5
+
+typedef enum {
+    kInviteRow = 0,
+    kPrivacyRow = 1,
+    kNotificationRow = 2,
+    kLinkedDevices = 3, // we don't actually use this, instead we segue via Interface Builder
+    kAdvancedRow = 4,
+    kAboutRow = 5,
+    kBlockListRow = 6,
+} kGeneralSectionRows;
```

Comment:
I was sorely tempted to modify this view to use a OWSTableSection, but I haven't (yet).